app/src/main/res/layout/fragment_map.xml

```diff
@@ -48,9 +48,32 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         android:text="@string/gps_accuracy_double_colon"
-        android:textColor="@color/red"
+        android:textColor="@color/toolbar"
         android:textSize="@dimen/text_size_twenty" />
 
+    <LinearLayout
```

Comment:
The parent is a Constraint layout. Can we remove this linear layout from the view and use constraints instead?

Reply:
Using a constraint layout with Group accomplishes the same desired effect. Updating the code.